app/livechat/client/views/app/tabbar/customerChatHistoryMessages.js

```diff
@@ -0,0 +1,43 @@
+import { Template } from 'meteor/templating';
+import moment from 'moment';
+import './customerChatHistoryMessages.html';
+import { ReactiveVar } from 'meteor/reactive-var';
+
+import { APIClient } from '../../../../../utils/client';
+
+Template.customerChatHistoryMessages.helpers({
+	historyResult() {
+		// will return all the messages in history room
+		return Template.instance().historyResult.get();
+	},
+});
+Template.customerChatHistoryMessages.onCreated(function() {
+	const currentData = Template.currentData();
+	this.historyResult = new ReactiveVar([]);
+	this.history = new ReactiveVar([]);
+	const id = currentData.clickRid;
+	const token = currentData.clickToken;
+	let closingDay;
+	let agentName;
+	this.autorun(async () => {
+		const { messages } = await APIClient.v1.get(`livechat/messages.history/${ id }?token=${ token }`);
+		// will return pervious chats list
+		this.history.set(messages);
+		const allMessages = [];
+		for (let j = messages.length - 2; j >= 1; j--) {
```

Comment:
You don't need to this here, you need to create a `customerChatHistoryMessagesItem` template and add a helper method there to display this kind of value.

Please, remove this loop and apply it in the right place.

Reply:
done